spec/apollo-federation/service_field_spec.rb

```diff
@@ -148,6 +148,135 @@ def execute_sdl(schema)
       )
     end
 
+    it 'returns valid SDL for shareable types' do
```

Comment:
Given that the spec cases are testing the same things working in Federation 2.0, I suggest that we make a separate `v2` folder for all the v2 test cases? The specs in this case can be named `spec/apollo-federation/v2/service_field_spec.rb`. 

Makes it easier to find them. When the day comes when we don't support 1.0 anymore, it makes clean up easier too.